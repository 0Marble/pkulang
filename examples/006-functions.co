fn foo() void {
	println("->foo");
	% we don't care about declaration order
	bar();
	println("foo->");
}

fn main() void {
	println("->main");
	foo();
	println("main->");
}

fn bar() void {
	println("->bar");
	println("bar->");
}
