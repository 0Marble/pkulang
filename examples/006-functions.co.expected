->main
->foo
->bar
bar->
foo->
main->
